Add infomration to use Source Side Validation script (#1476)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/collaboration-exo/public-folders/batch-migration-of-legacy-public-folders.md

You'll perform the migration using the **\*-MigrationBatch** cmdlets, in addition to the following PowerShell scripts:

> - ` ssv.ps1`:  Source Side Validation script scans the public folders at source and reports issues found along with action to fix the issues. You'll run this script on the legacy Exchange server On-Premises.

Reviewer: ```suggestion
- ` ssv.ps1`:  Source Side Validation script scans the public folders at source and reports issues found along with actions required to fix the issues. You'll run this script on the legacy Exchange server on-premises.
```

    - `SyncMailPublicFolders.strings.psd1`

> 4. Download the source side validation script from https://www.microsoft.com/en-us/download/confirmation.aspx?id=100414

Reviewer: ```suggestion
4. Download the source side validation script from the [Microsoft Download Center](https://www.microsoft.com/download/confirmation.aspx?id=100414). 
```

> 5. Save the scripts to the same location you did for step 2. For example, C:\PFScripts.

Reviewer: ```suggestion
5. Save the scripts to the same location as you did in step 2, for example, C:\PFScripts.
```

### Prerequisite steps on the legacy Exchange server

> Add an additional step here suggesting to run Source Side Validation script at mailbox server on Exchange Server 2010. Please use the examples as documented here: https://techcommunity.microsoft.com/t5/Exchange-Team-Blog/Making-your-public-folder-migrations-faster-and-more-reliable/ba-p/917622

Reviewer: ```suggestion
We recommend running Source Side Validation script from a mailbox server on Exchange Server 2010. Please use the examples as documented [here](https://techcommunity.microsoft.com/t5/Exchange-Team-Blog/Making-your-public-folder-migrations-faster-and-more-reliable/ba-p/917622).
```
